pkg/controller/conformance/resources.go

```diff
@@ -75,10 +74,6 @@ func (r *IntResource) SetValue(v int) {
 	r.value.value = v
 }
 
-func (r *IntResource) String() string {
```

Comment:
Does new implementation print enough info? I see `r.value.value` here - is it important?

Reply:
it was more of a test thing I guess (for debugging), used only in the tests